android/src/main/java/com/swmansion/gesturehandler/core/LongPressGestureHandler.kt

```diff
@@ -37,6 +40,37 @@ class LongPressGestureHandler(context: Context) : GestureHandler<LongPressGestur
     return this
   }
 
+  fun setNumberOfPointers(numberOfPointers: Int): LongPressGestureHandler {
+    numberOfPointersRequired = numberOfPointers
+    return this
+  }
+
+  private fun getAverageCoords(ev: MotionEvent, excludePointer: Boolean = false): Pair<Float, Float> {
+    if (!excludePointer) {
+      val x = (0 until ev.pointerCount).map { ev.getX(it) }.average().toFloat()
+      val y = (0 until ev.pointerCount).map { ev.getY(it) }.average().toFloat()
+
+      return Pair(x, y)
```

Comment:
```suggestion
      return x to y
```

Reply:
I'm no expert in Kotlin and using `Pair` seems cleaner to me (I mean it is easier to understand what's going on). I'll leave that as it is now.